test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -133,10 +165,21 @@ func testNvidiaGPUsOnCOS(f *framework.Framework) {
 		Skip("Nvidia GPU tests are supproted only on Container Optimized OS image currently")
 	}
 	framework.Logf("Cluster is running on COS. Proceeding with test")
+
+	if f.BaseName == "device-plugin-gpus" {
+		dsYamlUrl = "https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia_gpu/device-plugin-daemonset.yaml"
+		gpuResourceName = v1.ResourceOpaqueIntPrefix + "nvidiaGPU"
```

Comment:
Why are you using OIRs? That's deprecated.

Reply:
Fixed.